docker-compose.override.yml.example

```diff
@@ -53,6 +53,12 @@
 #   api:
 #     image: ghcr.io/danny-avila/librechat-api:latest
 
+# # ADD SAML METADATA FILE
+#   api:
+#     - type: bind
+#       source: ./your-metadata.xml
+#       target: /app/your-metadata.xml
+
```

Comment:
I think you missed here something? you mean the cert file here right?

Reply:
Yes

Reply:
I think you still made a typo 😅 `ycert.pem`

Reply:
Changed it to `your_cert.pem`